pre_commit/main.py

```diff
@@ -107,6 +107,17 @@ def _add_run_options(parser: argparse.ArgumentParser) -> None:
         '--commit-msg-filename',
         help='Filename to check when running during `commit-msg`',
     )
+    parser.add_argument(
+        '--commit-msg',
```

Comment:
this is a bad name for this -- perhaps `prepare-commit-message-source` ?  it doesn't contain the commit message itself so calling it `commit-msg` doesn't make sense